src/scalarstats.jl

```diff
@@ -648,12 +648,12 @@ Pretty-print the summary statistics provided by [`summarystats`](@ref):
 the mean, minimum, 25th percentile, median, 75th percentile, and
 maximum.
 """
-describe(a::AbstractArray) = describe(stdout, a)
-function describe(io::IO, a::AbstractArray{T}) where T<:Union{Real,Missing}
+DataAPI.describe(x) = describe(stdout, x)
```

Comment:
We should reexport `describe` to avoid breaking any packages which may be overloading it from StatsBase.